kinetic/distribution.yaml

```diff
@@ -6562,34 +6562,41 @@ repositories:
       url: https://github.com/ros-drivers/leap_motion.git
       version: hydro
     status: maintained
-  leo_description:
+  leo_common:
     doc:
       type: git
-      url: https://github.com/LeoRover/leo_description.git
+      url: https://github.com/LeoRover/leo_common.git
       version: master
     release:
+      packages:
+      - leo
+      - leo_description
+      - leo_teleop
       tags:
-        release: release/kinetic/{package}/{version}
-      url: https://github.com/fictionlab-gbp/leo_description-release.git
-      version: 0.3.0-1
+        release: release/melodic/{package}/{version}
```

Comment:
This doesn't seem right; this should be kinetic.  The same goes below.

Reply:
Sorry, I forgot to change the tags. Please check the newest commit.